defaults/annotations/annotations.go

```diff
@@ -0,0 +1,5 @@
+package annotations
```

Comment:
this may be a go formatting thing but, would it make more sense to have the package contain the word `default` so that when you import this, its super straightforward that's the only purpose of the package? i.e. what if there's some extension you need later that is for annotations. 

recommendation: all packages in defaults should be named `default<original package name>` i.e. `defaultannotations`. What do you all think?